tests/mockServer/mock.js

```diff
@@ -1190,20 +1187,19 @@ function putTags(req, res){
 
 function deleteTags(req, res){
   const collectedArtifact = artifacts.artifacts
-    .find(artifact => (((artifact.metadata?.project === req.params.project) 
-    || artifact.project === req.params.project) 
-    && artifact.kind === req.body.identifiers[0].kind
-    && ((artifact.metadata?.tree === req.body.identifiers[0].uid) 
-    || artifact.tree === req.body.identifiers[0].uid) 
-    && ((artifact.spec?.db_key === req.body.identifiers[0].key) 
-    || artifact.db_key === req.body.identifiers[0].key)))
-  if (collectedArtifact) {
-    if (collectedArtifact.metadata?.tag === req.params.tag){
-      collectedArtifact.metadata.tag = ''
-    }
-    else if (collectedArtifact.tag === req.params.tag){
-      collectedArtifact.tag = ''
-    }
+    .filter(artifact => ((artifact.metadata?.project === req.params.project || artifact.project === req.params.project) 
```

Comment:
define a const, like artiact = artifact.metadata ?? artifact adn use it where you need